unified_targets/configs/EXF722DUAL.config

```diff
@@ -13,40 +13,98 @@ resource MOTOR 5 B06
 resource MOTOR 6 B07
 resource MOTOR 7 B01
 resource MOTOR 8 B00
+resource SERVO 1 NONE
```

Comment:
All of the lines with `NONE`, or containing defaults should not be in here - apply this config to a unified target, then do a `diff` to get the lines that should be kept.

Reply:
I have forced pushing the code to the repository, thank you for your suggestion.